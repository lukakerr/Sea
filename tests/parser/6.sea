fn main {} (
  if {true}
    ret "ok"
  ) else (
    ret "fail"
  )
)