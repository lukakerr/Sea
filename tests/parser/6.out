Error: Expected run keyword at line 2
